plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer-v3.ts

```diff
@@ -244,19 +241,23 @@ export class SessionRecordingIngesterV3 {
                     },
                 })
 
-                // await runInstrumentedFunction({
-                //     statsKey: `recordingingester.handleEachBatch.consumeReplayEvents`,
-                //     func: async () => {
-                //         await this.replayEventsIngester.consumeBatch(recordingMessages)
-                //     },
-                // })
-
-                // await runInstrumentedFunction({
-                //     statsKey: `recordingingester.handleEachBatch.consumeConsoleLogEvents`,
-                //     func: async () => {
-                //         await this.consoleLogsIngester.consumeBatch(recordingMessages)
-                //     },
-                // })
+                if (this.replayEventsIngester) {
+                    await runInstrumentedFunction({
+                        statsKey: `recordingingester.handleEachBatch.consumeReplayEvents`,
+                        func: async () => {
+                            await this.replayEventsIngester!.consumeBatch(recordingMessages)
+                        },
+                    })
+                }
+
+                if (this.consoleLogsIngester) {
+                    await runInstrumentedFunction({
+                        statsKey: `recordingingester.handleEachBatch.consumeConsoleLogEvents`,
+                        func: async () => {
+                            await this.consoleLogsIngester!.consumeBatch(recordingMessages)
+                        },
+                    })
+                }
```

Comment:
we've already (I think) got counters in place so we can see which pod is handling these in grafana for safety ?

Reply:
Yes indeed we do. The two dashboards have separate metrics